### PR TITLE
chore(version): Bump version 1.7.0

### PR DESCRIPTION
**Pull request by Equartey**
chore(version): Bump version

### Features
- feat(api): add copyWith to GraphQLRequest ([#4365](https://github.com/aws-amplify/amplify-flutter/pull/4365))

### Fixes
- fix: `google.crypto.tink` version constraint ([#4434](https://github.com/aws-amplify/amplify-flutter/pull/4434))

### Chores
- chore(datastore): Amplify Swift version bump to 1.30.7 ([#4454](https://github.com/aws-amplify/amplify-flutter/pull/4454))

Updated-Components: amplify_lints, Amplify Flutter, Amplify Dart, Amplify UI, DB Common, Secure Storage, AWS Common, Smithy, Worker Bee

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml:1]: I think there should be changelog update and version.dart update for this package

File: packages/api/amplify_api_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/api/amplify_api_dart/pubspec.yaml:1]: this package is missing changelog update

File: packages/authenticator/amplify_authenticator/pubspec.yaml
Comment (NikaHsn) [inline packages/authenticator/amplify_authenticator/pubspec.yaml:1]: this packge is missing update for changelog and version.dart

File: packages/common/amplify_db_common_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/common/amplify_db_common_dart/pubspec.yaml:1]: this package is missing changelog update

File: packages/storage/amplify_storage_s3_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/storage/amplify_storage_s3_dart/pubspec.yaml:1]: this package is missing changelog update

**Review by NikaHsn — APPROVED** [review]

**Review by khatruong2009 — APPROVED** [review]

**Pull request merged**